routes/fallback.php

```diff
@@ -3,8 +3,8 @@
 
 if ($entry = Data::findByRequestUrl(request()->url())) {
     if (View::exists($entry->get('template'))) {
-        echo view($entry->get('template'), $entry->data());
+        echo view($entry->get('template'), $entry);
     } else if (View::exists(str($entry->structure()->handle())->singular()->toString())) {
-        echo view(str($entry->structure()->handle())->singular()->toString(), $entry->data());
+        echo view(str($entry->structure()->handle())->singular()->toString(), $entry);
```

Comment:
`$entry->data()` grabbed all the raw data without parsing. passing $entry to the view is exactly the same, but strings as `entry::uuid-asdf-fdjakl` will be parsed to `/url`.